File: 2 fibonacci/FiboLoop.java
public class FiboLoop {
    public static void main(String[] args) {
        int first = 0;
        int second = 1;

        System.out.println(first);
        System.out.println(second);

        for (int i = 0; i<= 18; i++) {
            int cur = first + second;
            System.out.println(cur);
            second = first;
            first = cur;
        }
    }
}

File: 2 fibonacci/fiboLoop.cpp
#include <iostream>

int main()
{
    int first = 0;
    int second = 1;

    std::cout << first << std::endl;
    std::cout << second << std::endl;

    for (int i = 0; i < 19; i++)
    {
        int cur = first + second;
        std::cout << cur << std::endl;
        second = first;
        first = cur;
    }
}